scripts/create-local-queues.sh

```diff
@@ -44,7 +44,7 @@ OUTPUT_QUEUE_URL_LOCALHOST=${OUTPUT_QUEUE_URL/sqs.eu-west-1.localhost.localstack
 
 echo "Created output queue in localstack, url: ${OUTPUT_QUEUE_URL_LOCALHOST}"
 
-MEDIA_DOWNLOAD_QUEUE_URL=$(aws --endpoint-url=http://localhost:4566 sqs create-queue --queue-name=$APP_NAME-media-download-queue-DEV | jq .QueueUrl)
+MEDIA_DOWNLOAD_QUEUE_URL=$(aws --endpoint-url=http://localhost:4566 sqs create-queue --queue-name=$APP_NAME-media-download-queue-DEV.fifo --attributes "FifoQueue=true,ContentBasedDeduplication=true" | jq .QueueUrl)
```

Comment:
why does the queue need to be FIFO?

Reply:
In this case I'm just bringing it in line with the existing infra (which was already fifo - so the local infra should have been https://github.com/guardian/transcription-service/pull/176/files#diff-4c04d9d03ec555fc8ffc3cdb2814f0e13bb54815a58ab0160ed1182e272ed9a5) but I think that it shouldn't be FIFO really, something to change in a follow up bit of work